packages/react-scripts/scripts/utils/verifyTypeScriptSetup.js

```diff
@@ -165,6 +164,7 @@ function verifyTypeScriptSetup() {
     getNewLine: () => os.EOL,
   };
 
+  const errors = [];
```

Comment:
All the changes I've made in this file should equally be applied to any `jsonconfig.json` that is being used. However, there is no file that does at the moment. Should we have a `verifyJavaScriptSetup.js`? 👀